subprojects/gst-plugins-base/gst-libs/gst/sdp/gstsdpmessage.c

```diff
@@ -3642,6 +3642,9 @@ gst_sdp_media_get_caps_from_media (const GstSDPMedia * media, gint pt)
     /* we fail if we cannot find one */
     if (rate == -1)
       goto no_rate;
+
+    if (rate == 0 && name != NULL && (!g_strcmp0(name,"h264") || !g_strcmp0(name,"H264")))
+      rate = 90000;
```

Comment:
it is strange that the code arrives here with rate == 0 AND codec name == "h264", because in the lines above it should have been received the rate from [gst_rtp_payload_info_for_name](https://github.com/vicon-security/gstreamer/blob/cemtrex-1.20/subprojects/gst-plugins-base/gst-libs/gst/rtp/gstrtppayloads.c#L205) .

P.S. But gst_rtp_payload_info_for_name works only if media->media == "video".
@deepu-abraham can you please check what is the value of media->media here? Also, what is the value of the "pt" ?

Reply:
also, just to mention,
```
name != NULL
```
comparison is not needed because g_strcmp0 already checks it's parameters for NULL (unlike strcmp).